include/vkgcDefs.h

```diff
@@ -81,8 +81,10 @@
 //  @page VersionHistory
 //  %Version History
 //  | %Version | Change Description                                                                                    |
-//  | -------- | ----------------------------------------------------------------------------------------------------- |
-//  |     54.0 | Add overrideThreadGroupSizeX,overrideThreadGroupSizeY and overrideThreadGroupSizeZ to PipelineOptions |
+//  | -------- | ---------------------------------------------------------------------------------------------|
+//  |     54.1 | Add overrideForceThreadIdSwizzling overrideShaderThreadGroupSizeX, overrideShaderThreadGroupSizeY 
+//  |          |and overrideShaderThreadGroupSizeZ  to PipelineShaderOptions     |
+//  |     54.0 | Add overrideThreadGroupSizeX, overrideThreadGroupSizeY and overrideThreadGroupSizeZ  to PipelineOptions     |
```

Comment:
The formatting look a little off.  The `|` at the end of the lines do not seem to line up.